base/expr.jl

```diff
@@ -2,12 +2,12 @@
 
 ## symbols ##
 
-symbol(s::Symbol) = s
-symbol(s::ASCIIString) = symbol(s.data)
-symbol(s::UTF8String) = symbol(s.data)
-symbol(a::Array{UInt8,1}) =
+Symbol(s::Symbol) = s
```

Comment:
Is this what's causing the method error(s) @tkelman noticed?

...but like I say removing the convert kills the build. :/